src/containers/howitworks/HowItWorksSection.js

```diff
@@ -0,0 +1,48 @@
+import React, { Fragment } from 'react';
+
+import styled from 'styled-components';
+
+import PageSection from '../../components/layout/PageSection';
+import StyledSectionSubTitle from '../../components/headers/StyledSectionSubTitle';
+import StyledSectionTitle from '../../components/headers/StyledSectionTitle';
+import { MEDIA_QUERY_LG } from '../../core/style/Sizes';
+import { TITLE, SUBTITLE } from './Constants';
+import TabDisplay from './TabDisplay';
+
+import { N1 } from '../../core/style/Colors';
+
+const Content = styled.div`
+  margin: 80px 0 100px 0;
+  max-width: 100%;
+
+  @media only screen and (min-width: ${MEDIA_QUERY_LG}px) {
+    display: block;
+    margin: 140px 0 140px 0;
+  }
+`;
+
+const StyledCroppedSubtitle = StyledSectionSubTitle.extend`
+  margin-bottom: 60px;
+
+  @media only screen and (min-width: ${MEDIA_QUERY_LG}px) {
+    width: 46%;
+  }
+`;
+
+const HowItWorksSection = () => (
```

Comment:
The "How It Works" section looks pretty bad when you resize the screen to below 1020px. The image basically becomes full screen.

![screen shot 2018-06-25 at 10 05 15 am](https://user-images.githubusercontent.com/440299/41864566-4d77ac2c-785f-11e8-810b-0c7f68f70ac9.png)

Reply:
Fixed, but probably needs some design in put from Paul (both small and med browser sizes, and for everything)